### PR TITLE
Updates to pattern library for status and moving components

### PR DESCRIPTION
**Pull request by line47**
Moved some more complex items under features like card and panel. Also
updated and added some basic read for actions.

### CODE REVIEW

File: docs/src/components/components/action/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+# Action
+
+The **action** component is something to use when you want the user to take an
+action. Buttons should be used for changing the state of a page or transforming
+data. Links should be used for navigation.
+
+## Use
+
+To use the **action** component, add the class `action` to a button or add the
```
Comment (thisisdano) [inline docs/src/components/components/action/README.md]: Worth being specific that this needs to go on a `<button>` element — this is _almost_ specific enough! :)

File: docs/src/main.css [hunk 2]
```diff
@@ -378,7 +379,6 @@ body._hasBackground {
 .Prose code pre {
   font-family: 'Roboto Mono';
   background-color: transparent;
-  width: 100%;
   display: block;
 }
```
Comment (thisisdano) [inline docs/src/main.css:384]: These last changes _probably_ screw up some of my utility class docs (I'd made these changes to make the code blocks in that doc look right...)

**Review by thisisdano — COMMENTED** [review]
Couple quick things...

File: docs/src/main.css (hunk 2 above)
Comment (line47) [reply docs/src/main.css:384]: The 100% width shouldn't be needed as display:block does that for us. Looks good on my local. Can you verify locally?

File: docs/src/components/components/action/README.md (hunk 1 above)
Comment (line47) [reply docs/src/components/components/action/README.md]: Good call, I'll update.

File: docs/src/main.css (hunk 2 above)
Comment (thisisdano) [reply docs/src/main.css:384]: If it looks good to you, that's good enough for me.

**Review by thisisdano — APPROVED** [review]

**Pull request merged**